code-challenge/src/components/ArticleList/ArticleList.scss

```diff
@@ -0,0 +1,4 @@
+.ArticleList {
+  width: 50%;
```

Comment:
To get this looking a little more functional on mobile we could remove the 50% width rule
```suggestion
```
…and add the following to the Toolbar component:
```
flex-wrap: wrap;
justify-content: space-between;
```
<img width="394" alt="Screen Shot 2021-08-06 at 12 32 08 PM" src="https://user-images.githubusercontent.com/211766/128562914-36a782b9-2fef-4d44-933f-b1497226de6b.png">

Reply:
good call, my next commit should make all of this look better

Reply:
that will be on my tombstone